projects/extension/src/components/BraveModal.tsx

```diff
@@ -91,10 +84,9 @@ export const BraveModal = ({ show, isOptions }: Props) => {
             <button
               type="button"
               onClick={() => {
+                // TODO: this should produce a react-style event instead of setting the value directly
```

Comment:
I am not sure I understand what a "react-style event" is. Setting the state of a State variable is the react approach of triggering a re-render of the virtual dom.

Reply:
The `BraveModal` should have a `onClose` property as a callback, and it's the user of the `BraveModal` that sets the value in local storage, instead of the `BraveModal` itself directly setting the value in local storage